mscore/palettebox.cpp

```diff
@@ -63,6 +69,17 @@ PaletteBox::PaletteBox(QWidget* parent)
       hlSearch->setContentsMargins(5,0,5,0);
       hlSearch->addWidget(_searchBox);
 
+
+      QFile file(":/default.txt");
+      file.open(QIODevice::ReadOnly);
+      TreeModel* model = new TreeModel(file.readAll(),this);
```

Comment:
You're going to need to access the model in other functions within the PaletteBox class, so you will have to declare it first in palettebox.h [as is done for the searchbox](https://github.com/musescore/MuseScore/blob/f0768f1633473e31571299eba7295810dcc871de/mscore/palettebox.h#L34) before you start using it here.

Reply:
That will enable you to start adding TreeItems to the model in [PaletteBox::read](https://github.com/musescore/MuseScore/blob/f0768f1633473e31571299eba7295810dcc871de/mscore/palettebox.cpp#L418).